.github/workflows/sonarqube.yml

```diff
@@ -3,13 +3,6 @@ on:
     branches:
       - master
       - sq-10
-    paths-ignore:
-      - '.github/actions/**'
-  pull_request_target:
```

Comment:
as underlying action is entirely removed, we have no need for the pull request target logic.